launch/interaction/speech_recognition.launch

```diff
@@ -3,7 +3,15 @@
 
 	<!-- Get the machine file -->
 	<arg name="machine" default="localhost"/>
-        <arg name="dragonfly" default="true" />
+    <arg name="dragonfly" default="true" />
+    <arg name="kaldi" default="false" />
+
+    <group if="$(arg kaldi)">
+        <!-- Kaldi model path default to ~/data/kaldi_speech_recognition -->
+        <arg name="model_path" default="$(env HOME)/data/speech_models/model/"/>
+        <param name="/kaldi_model_path" type="string" value="$(arg model_path)"/>
```

Comment:
@ar13pit why the absolute param path? This shouldn't be the case in my opinion

Reply:
In the `install.bash` file of the target, I download the trained model into this directory that's why I put it up this way.

If you have a better way of using this parameter then lets use that

Reply:
No it is about the param **name**.  This is not a param which is used by multiple nodes. So why is it a global param? I would like to see it as an private parameter.

Reply:
Definitely. It needs to be private and not global.

Reply:
@ar13pit Do you apply the change? This maybe also requires a change in the node

Reply:
You make the appropriate change here and I'll fix the node after that

Reply:
Done